R/carbon_analysis.R

```diff
@@ -0,0 +1,72 @@
+## pull in data for carbon lakes analysis
```

Comment:
If you can become an `lapply` expert, more power to you. That would be great. But for me....I've acknowledge my R shortcomings and reconciled that I'll never fully understand them, and therefore stick to old fashioned loops.